kimyeongseok/app.js

```diff
@@ -0,0 +1,60 @@
+const http = require ("http");
+const express = require ("express");
+const cors = require ("cors");
+const morgan = require ("morgan");
+
+const dotenv = require ("dotenv");
+dotenv.config()
+
+const { DataSource } = require('typeorm')
+
+const myDataSource = new DataSource({
```

Comment:
```suggestion
const appDataSource = new DataSource({
```

myDataSource 보다는 appDataSource , mysqlDataSource 혹은 간결하게 database로 변수명을 변경해주세요. “my”라는 용어는 예제 코드에서는 사용할 수 있으나 실제 코드에서는 피해주세요.